starkinfra/issuingtoken/__issuingtoken.py

```diff
@@ -0,0 +1,239 @@
+from json import dumps
+from starkinfra.utils import rest
+from starkcore.utils.api import api_json
+from ..utils.parse import parse_and_verify
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date
+
+
+
+class IssuingToken(Resource):
+    """# IssuingToken object
+    The IssuingToken object displays the information of the tokens created in your Workspace.
+    ## Attributes (return-only):
+    - card_id [string]: card ID which the token is bounded to. ex: "5656565656565656"
+    - wallet_id [string]: wallet provider which the token is bounded to. ex: "google"
+    - wallet_name [string]: wallet name. ex: "GOOGLE"
+    - merchant_id [string]: merchant unique id. ex: "5656565656565656"
+    ## Attributes (IssuingToken only):
+    - id [string]: unique id returned when IssuingToken is created. ex: "5656565656565656"
+    - created [datetime.datetime]: creation datetime for the IssuingToken. ex: datetime.datetime(2020, 3, 10, 10, 30, 0, 0)
+    - updated [datetime.datetime]: latest update datetime for the IssuingToken. ex: datetime.datetime(2020, 3, 10, 10, 30, 0, 0)
+    - external_id [string]: a unique string among all your IssuingTokens, used to avoid resource duplication. ex: "DSHRMC00002626944b0e3b539d4d459281bdba90c2588791"
+    - status [string]: current IssuingToken status. ex: "active", "blocked", "canceled", "frozen" or "pending"
+    - tags [list of strings]: list of strings for reference when searching for IssuingToken. ex: ["employees", "monthly"]
+    ## Attributes (authorization request only):
+    - activation_code [string]: activation code recived through the bank app or sms. ex: "481632" 
+    - method_code [string]: provisioning method. Options: "app", "token", "manual", "server" or "browser"
+    - device_type [string]: device type used for tokenization. ex: "Phone"
+    - device_name [string]: device name used for tokenization. ex: "My phone" 
+    - device_serial_number [string]: device serial number used for tokenization. ex: "2F6D63"
+    - device_os_name [string]: device operational system name used for tokenization. ex: "Android"
+    - device_os_version [string]: device operational system version used for tokenization. ex: "4.4.4"
+    - device_imei [string]: device imei used for tokenization. ex: "352099001761481"
+    - wallet_instance_id [string]: unique id refered to the wallet app in the current device. ex: "71583be4777eb89aaf0345eebeb82594f096615ed17862d0"
+    """
```

Comment:
Reordenei pra deixar no padrão do SDK, arruma a ordem dos parâmetros do construtor

```md
# IssuingToken object
The IssuingToken object displays the information of the tokens created in your Workspace.
## Attributes (return-only):
- card_id [string]: card ID which the token is bounded to. ex: "5656565656565656"
- wallet_id [string]: wallet provider which the token is bounded to. ex: "google"
- wallet_name [string]: wallet name. ex: "GOOGLE"
- merchant_id [string]: merchant unique id. ex: "5656565656565656"
## Attributes (IssuingToken only):
- id [string]: unique id returned when IssuingToken is created. ex: "5656565656565656"
- external_id [string]: a unique string among all your IssuingTokens, used to avoid resource duplication. ex: "DSHRMC00002626944b0e3b539d4d459281bdba90c2588791"
- tags [list of strings]: list of strings for reference when searching for IssuingToken. ex: ["employees", "monthly"]
- status [string]: current IssuingToken status. ex: "active", "blocked", "canceled", "frozen" or "pending"
- updated [datetime.datetime]: latest update datetime for the IssuingToken. ex: datetime.datetime(2020, 3, 10, 10, 30, 0, 0)
- created [datetime.datetime]: creation datetime for the IssuingToken. ex: datetime.datetime(2020, 3, 10, 10, 30, 0, 0)
## Attributes (authorization request only):
- activation_code [string]: activation code recived through the bank app or sms. ex: "481632" 
- method_code [string]: provisioning method. Options: "app", "token", "manual", "server" or "browser"
- device_type [string]: device type used for tokenization. ex: "Phone"
- device_name [string]: device name used for tokenization. ex: "My phone" 
- device_serial_number [string]: device serial number used for tokenization. ex: "2F6D63"
- device_os_name [string]: device operational system name used for tokenization. ex: "Android"
- device_os_version [string]: device operational system version used for tokenization. ex: "4.4.4"
- device_imei [string]: device imei used for tokenization. ex: "352099001761481"
- wallet_instance_id [string]: unique id refered to the wallet app in the current device. ex: "71583be4777eb89aaf0345eebeb82594f096615ed17862d0"
```